javav2/example_code/elasticbeanstalk/src/main/java/com/aws/example/DeleteApplication.java

```diff
@@ -0,0 +1,73 @@
+//snippet-sourcedescription:[DeleteApplication.java demonstrates how to delete an AWS Elastic Beanstalk application .]

```

Comment:
Remove extra space between "application" and the ending punctuation.

Reply:
fixed